frontend/src/app/[locale]/events/EventsHero.tsx

```diff
@@ -10,32 +10,28 @@ export default function EventsHero() {
   return (
     <div
       data-testid="events-hero"
-      className="events-hero bg-primary-darkest text-white padding-left-6 padding-y-2"
+      className="text-white bg-primary-darkest padding-top-4 tablet:padding-y-6"
     >
       <GridContainer>
-        <Grid row gap={3} className="margin-top-6">
-          <Grid
-            tablet={{
-              col: true,
-            }}
-          >
-            <h1 className="tablet:font-sans-2xl desktop-lg:font-sans-3xl desktop-lg:margin-top-2 text-balance">
-              {t("header")}
-            </h1>
-            <p className="usa-intro font-sans-md tablet:font-sans-lg line-height-sans-4 text-balance">
+        <Grid row gap>
+          <Grid tablet={{ col: "fill" }}>
+            <h1 className="desktop-lg:font-sans-3xl">{t("header")}</h1>
+            <p className="text-balance font-sans-md tablet:font-sans-lg margin-bottom-4 tablet:margin-bottom-0">
```

Comment:
- We don't need the `margin-top` on the grid itself; container handles this
- Set text column to fill available space (image column is responsive)
- Make h1 larger than default 2xl size only on desktop (headings now get global styles; no need to define multiple breakpoint sizes)
- Remove unnecessary `.usa-intro` class (it's not doing anything since paragraphs now get global styles, but we do still want to define breakpoint sizes here so the homepage hero looks rad)